constraint_utils/constraint_basics.py

```diff
@@ -355,6 +395,14 @@ def get_proportion_observed_by_coverage(exome_ht: hl.Table, context_ht: hl.Table
     freq_index = exome_ht.freq_index_dict.collect()[0][dataset]
 
     def keep_criteria(ht):
+        """
+        Generate expression to keep pass variants with an allele count greater than 0.
```

Comment:
```suggestion
        Generate expression to keep PASS variants with an allele count greater than 0.
```

Reply:
Just for curious, why should pass be capitalized?

Reply:
You don't really need to, it's just in the VCF filters section if it passes filters I think it has "PASS"